<launch>

    <node pkg="pulson_ros" name="reset" type="reset" output="screen">
        <param name="node_id" value="104" />
	<param name="ip_address" value="0" />
	<param name="serial_dev" value="/dev/clearpath/p440_002" />
        <param name="dev_path" value="" />
    </node>

</launch>
